Tests/UnitTests/Caching/DeviceCacheTests.swift

```diff
@@ -474,6 +483,292 @@ class DeviceCacheTests: TestCase {
         expect(self.deviceCache.isProductEntitlementMappingCacheStale) == true
     }
 
+    // MARK: - Virtual Currencies
```

Comment:
Not a blocker and can be done in the future! but I think we should try to split these huge test files into multiple files... These `MARK` sections are probably where I would start.